designer/apps/branding/migrations/0003_auto_20191022_1338.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-10-22 13:38
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('branding', '0002_auto_20190703_1512'),
```

Comment:
i'm wondering if this should be made to depend on 0017_auto_20191022_1252.py explicitly, to make sure we copy the values over before we remove the fields. also, it would be nice to name these migration files so that it is clearer what they are doing, since they are doing more than the standard.

Reply:
@brittneyexline I added that dependency to this migration, and renamed the new migration files to be descriptive about what they do 👍